framework/core/src/Discussion/Discussion.php

```diff
@@ -224,8 +225,8 @@ public function setLastPost(Post $post)
      */
     public function refreshLastPost()
     {
-        /** @var Post $lastPost */
         if ($lastPost = $this->comments()->latest()->first()) {
+            /** @var Post $lastPost */
             $this->setLastPost($lastPost);
```

Comment:
nit: `latestPost` feels like better naming.

Reply:
Also, is the relationship not typed on the Eloquent level?

Reply:
> nit: `latestPost` feels like better naming.

in what sense? the last post is referred to last post everywhere.

Reply:
Nevermind, didn't look into the context.